core/src/main/kotlin/org/evomaster/core/problem/rest/seeding/postman/pojos/Info.kt

```diff
@@ -0,0 +1,7 @@
+package org.evomaster.core.problem.rest.seeding.postman.pojos
+
+data class Info (
+        val _postman_id: String,
```

Comment:
Just because Postman automatically sets a JSON property called `_postman_id`. It's easier to call the POJOs' attributes the same as they are called in the JSON, otherwise Gson or Jackson won't deserialize those fields correctly. I guess there should be ways to call the attributes differently than the JSON properties and have Gson/Jackson take that into account, but I haven't looked into it.

Let me know if you're ok with this of prefer me to investigate it a bit.

Reply:
ah, i see. it's OK ;)